api/src/test/java/org/pmiops/workbench/rdr/RdrExportServiceImplTest.java

```diff
@@ -215,6 +221,25 @@ public void exportWorkspace_FocusOnUnderservedPopulation() throws ApiException {
     verify(mockRdrApi).exportWorkspaces(Arrays.asList(rdrWorkspace));
   }
 
+  @Test
+  public void exportWorkspace_CreatorInformation() throws ApiException {
+    List<Long> workspaceID = new ArrayList<>();
+    workspaceID.add(1l);
+    rdrExportService.exportWorkspaces(workspaceID);
+    verify(rdrExportDao, times(1)).save(anyList());
+
+    RdrWorkspace rdrWorkspace = toDefaultRdrWorkspace(mockWorkspace);
+    verify(mockRdrApi).exportWorkspaces(Arrays.asList(rdrWorkspace));
+
+    workspaceID = new ArrayList<>();
+    workspaceID.add(3l);
+    rdrExportService.exportWorkspaces(workspaceID);
+    rdrWorkspace = toDefaultRdrWorkspace(mockCreatorWorkspace);
+    rdrWorkspace.setCreator(
+        new RdrWorkspaceCreator().userId(2l).familyName("email").givenName("icannothas"));
```

Comment:
With fluent constructsors or builders, please break before every new parameter.

Reply:
so spotlessApply is putting them in one line. i had it like ....
            .userId(2l)
            .familyName("email")
            .givenName("icannothas"));

Reply:
ok. Spotless isn't the best poet.